data/plugins/skill/mining/gem.rb

```diff
@@ -14,7 +14,7 @@ def gem(gem)
   GEMSTONES[gem.id] = gem
 end
 
-gem(Gemstone.new(1623, 0)) # uncut sapphire
-gem(Gemstone.new(1605, 0)) # uncut emerald
-gem(Gemstone.new(1619, 0)) # uncut ruby
-gem(Gemstone.new(1617, 0)) # uncut diamond
+gem(Gemstone.new(1623, 7)) # uncut sapphire
+gem(Gemstone.new(1605, 5)) # uncut emerald
+gem(Gemstone.new(1619, 3)) # uncut ruby
+gem(Gemstone.new(1617, 1)) # uncut diamond
```

Comment:
Can we change this method to take a symbol for the name (and use `lookup_item`) and a hash for the chance, e.g.:

```ruby
gem :uncut_sapphire, chance: 7
gem :uncut_emerald, chance: 5
gem :uncut_ruby, chance: 3
gem :uncut_diamond, chance: 1
```

That way we can get rid of the comments too